plutus-benchmark/nofib/test/formulaBudget.budget.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 8791690908
-| mem: 36693948})
\ No newline at end of file
+({cpu: 9642046908
+| mem: 40391148})
```

Comment:
I've doubled checked, this change is NOT caused by the latest change in `callSiteInline`, it was from the rebasing on master.